dist/src/main/config/broker.yaml.template

```diff
@@ -30,6 +30,10 @@
 # Relative paths are resolved relative to the installation directory of the broker.
 # ----------------------------------------------------
 
+# Enable the Camunda REST API to start on broker startup.
+# This setting can also be overridden using the environment variable CAMUNDA_REST_ENABLED.
+# camunda.rest.enabled: false
```

Comment:
```suggestion
# camunda.rest.enabled: true 
```
We normally show the default

Reply:
Ah, I assumed that that specific configuration template shows how to configure the Broker without an embedded gateway, that's what it also saying at the top of the file:
```
# Zeebe Broker configuration file (without embedded gateway)
```

That's why I added it here and set it to `false`. Do you want me to change that?